nixops/backends/gce.py

```diff
@@ -769,6 +780,25 @@ def backup(self, defn, backup_id):
                                     .format(volume.name, self.machine_name)
                 })
 
+            # Apply labels to snapshot just created
+            def check_snapshot_initiated():
```

Comment:
should be probably a wait until expected state e.g snapshot initiated ?

Reply:
i did that @AmineChikhaoui , I guess the expected state for when a snap was initiated (present) should be one of "UPLOADING", "CREATING" or "READY" (https://cloud.google.com/compute/docs/reference/rest/beta/snapshots). That being said, I think it takes only 1 or 2 seconds for that so the output would be something like this:
`$ nixops backup -d macys-dev-1
frontend.> backing up GCE machine 'n-b963de1933f911e89f780a29f7962a18-frontend' using ID '20180522045235'
frontend.> initiating snapshotting of disk 'n-b963de1933f911e89f780a29f7962a18-frontend-root': 'backup-20180522045235-e89f780a29f7962a18-frontend-root'
frontend.> . done
frontend.> updating labels of snapshot backup-20180522045235-e89f780a29f7962a18-frontend-root
frontend.> initiating snapshotting of disk 'n-b963de1933f911e89f780a29f7962a18-frontend-data': 'backup-20180522045235-e89f780a29f7962a18-frontend-data'
frontend.> . done
frontend.> updating labels of snapshot backup-20180522045235-e89f780a29f7962a18-frontend-data
20180522045235`
Let me know if you think that should be changed some other way, thanks !!